src/main/draft/tomcat.yaml

```diff
@@ -0,0 +1,457 @@
+        Fn::ImportValue:
+          !Sub "${vpcStackName}-bastion-security-group"
+
+  cassandra9042Ingress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId:
+        Fn::ImportValue:
+          !Sub "${cassandraStackName}-security-group"
+      IpProtocol: tcp
+      FromPort: 9042
+      ToPort: 9042
+      SourceSecurityGroupId: !Ref securityGroup
+
+  cassandra9042Ingress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId:
+        Fn::ImportValue:
+          !Sub "${cassandraStackName}-security-group"
+      IpProtocol: tcp
+      FromPort: 9160
+      ToPort: 9160
+      SourceSecurityGroupId: !Ref securityGroup
+
+  redis6379Ingress:
+    Type: AWS::EC2::SecurityGroupIngress
+    Properties:
+      GroupId:
+        Fn::ImportValue:
+          !Sub "${redisStackName}-security-group"
+      IpProtocol: tcp
+      FromPort: 6379
+      ToPort: 6379
+      SourceSecurityGroupId: !Ref securityGroup
+
+  deployableBucketReadRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - ec2.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      Policies:
+        - PolicyName: "DeployableBucketReadPolicy"
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:GetObject
+                  - s3:ListBucket
+                Resource:
+                  - !Join
+                    - ""
+                    - - "arn:aws:s3:::"
+                      - !Ref deployableBucketName
+                      - "/*"
+                  - !Join
+                    - ""
+                    - - "arn:aws:s3:::"
+                      - !Ref deployableBucketName
+
+  instanceProfile:
+    Type: AWS::IAM::InstanceProfile
+    Properties:
+      Path: /
+      Roles:
+        - !Ref deployableBucketReadRole
+
+  targetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      HealthCheckEnabled: true
+      HealthCheckIntervalSeconds: 10
+      HealthCheckPath: !Sub "/${applicationContextPath}/api/healthcheck"
+      HealthCheckTimeoutSeconds: 5
+      HealthyThresholdCount: 3
+      Port: 8080
+      Protocol: HTTP
+      TargetType: instance
+      UnhealthyThresholdCount: 2
+      VpcId:
+        Fn::ImportValue:
+          !Sub "${vpcStackName}-vpc"
+
+  listenerRule:
+    Type: AWS::ElasticLoadBalancingV2::ListenerRule
+    Properties:
+      Actions:
+        - Type: forward
+          TargetGroupArn: !Ref targetGroup
+      Conditions:
+        - Field: path-pattern
+          Values:
+            - !Sub "/${applicationContextPath}/*"
+      ListenerArn:
+        Fn::ImportValue:
+          !Sub "${publicLoadBalancerStackName}-public-lb-listener"
+      Priority: 10000
```

Comment:
Although this template is meant to be a draft and clearly states that you have to customize it to your needs, I wonder if we should make this a template parameter, to emphasize (or even explain in the description) that the value _has_ to be changed (unless your tomcat application is the only one on the given load balancer).

Thoughts?

Reply:
If someone only uses this pseudo template as it is, to deploy one application, this number does not matter. If he deploys a second, he will get errors. Maybe a comment?

Reply:
As discussed in person: a comment is as visible or invisible as the value itself when someone customizes this template. When multiple templates with the same priority are about to be deployed, the DevOps will get an error and the author will then remember to change this value.

I don't think we need a comment.